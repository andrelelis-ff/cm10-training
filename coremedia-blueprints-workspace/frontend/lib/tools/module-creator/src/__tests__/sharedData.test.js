"use strict";

const { initPrettierrc } = require("../sharedData");

describe("initPrettierrc()", () => {
  it("returns data for .prettierrc", () => {
    expect(initPrettierrc()).toMatchSnapshot();
  });
});
