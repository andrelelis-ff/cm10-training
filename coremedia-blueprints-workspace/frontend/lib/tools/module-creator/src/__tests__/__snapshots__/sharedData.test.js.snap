// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initPrettierrc() returns data for .prettierrc 1`] = `
"{
  \\"trailingComma\\": \\"es5\\"
}
"
`;
